File: src/main/java/org/jetbrains/pmdkotlin/lang/kotlin/ast/KotlinParserVisitorAdapter.java
package org.jetbrains.pmdkotlin.lang.kotlin.ast;

import com.intellij.openapi.progress.ProgressIndicatorProvider;
import com.intellij.psi.PsiElement;
import org.jetbrains.kotlin.psi.JetTreeVisitor;
import org.jetbrains.kotlin.psi.JetVisitor;

public class KotlinParserVisitorAdapter extends JetTreeVisitor implements KotlinParserVisitor {
}

File: src/main/kotlin/org/jetbrains/pmdkotlin/lang/kotlin/ast/KotlinParserVisitorAdapter.kt
package org.jetbrains.pmdkotlin.lang.kotlin.ast;

import net.sourceforge.pmd.lang.ast.Node

public open class KotlinParserVisitorAdapter: KotlinParserVisitor {
    public fun visit(node: Node, data: Any): Any? {
        (node as AbstractKotlinNode).childrenAccept(this, data)
        return data
    }
}



